examples/kitchen-sink.yaml

```diff
@@ -134,6 +134,12 @@ meter_provider:
             without_type_suffix: false
             # Configure Prometheus Exporter to produce metrics without a scope info metric.
             without_scope_info: false
+            # Configure Prometheus Exporter to add resource attributes as metrics attributes.
+            with_resource_constant_labels:
+              included:
```

Comment:
Can you add comments describing the semantics of these properties? The `kitchen-sink.yaml` is the best we currently have for describing behavioral expectations of implementations.

Reply:
Updated